atomata/capstone_WebClient
# Added new unity build folders + connected react buttons to unity

This PR : 

adds the updated unity build folder ( no buttons in unity scene + assetbundles loading)
connects the react buttons to unity so the apparatus can be loaded and the triggers can be called on the apparatus

To test : 

Run the app, click on wobble sphere or earth button on the apparatus list. They should properly load. For the wobble sphere, try the triggers (still, wobble, bounce)
